src/components/Form/Section.tsx

```diff
@@ -32,9 +35,8 @@ const StyledDescription = styled.p`
 
 const StyledRows = styled.div`
 	display: grid;
-	row-gap: 20px;
 	margin: 0px;
-	padding: ${pr => !pr.hasTitle ? "0px" : "0px 40px 50px 40px"} ;
+	padding: ${pr => pr.view === "MOBILE" ? "0px 30px 30px 30px" : "0px 40px 50px 40px"};

```

Comment:
I think this should go directly in the Row component and not here, because then it adds even more spacing to fields that are not in sections.

Reply:
On Row.tsx, the margin should now be 0px -15px because our fields are now 15px in padding. This value should be in theme.ts